guides/using-helpers.md

```diff
@@ -4,4 +4,166 @@ layout: guides
 permalink: /guides/using-helpers/
 ---
 
-#Using Helpers
+# Using Helpers
+
+Helpers are extensions added to Dust to increase its functionality. They make it possible for anyone to extend the way Dust works. They are commonly used to add functionality related to things like logic, formatting, and internationalization. Helpers are denoted by the `@` symbol. Let's take a look at some commonly used helpers.
+
+## Installing Helpers
+
+Helpers are bundled separately as the `dustjs-helpers` library. After the dustjs-linkedin library is loaded, follow the same instructions in the [Getting Started guide](/guides/getting-started/) to install `dustjs-helpers`. If you are cloning or downloading from GitHub, you can find the helpers in the [dustjs-helpers repository](https://github.com/linkedin/dustjs-helpers).
```

Comment:
Don't worry about adding it to the Getting Started Guide. I am making some big changes to that page, so I'll just add that as well.

The difference between dust-core and dust-full is that dust-full has the ability to parse and compile templates, but dust-core can only render already compiled templates.